cmd/helm/root.go

```diff
@@ -67,6 +67,7 @@ Environment variables:
 | $HELM_KUBEASUSER                   | set the Username to impersonate for the operation.                                |
 | $HELM_KUBECONTEXT                  | set the name of the kubeconfig context.                                           |
 | $HELM_KUBETOKEN                    | set the Bearer KubeToken used for authentication.                                 |
+| $HELM_DEFAULT_BURST_LIMIT          | set the default burst limit in the case the server contains many CRDs             |
```

Comment:
HELM_DEFAULT_BURST_LIMIT  --> HELM_BURST_LIMIT is right?

Reply:
This has been fixed.